values.yaml

```diff
@@ -64,6 +64,9 @@ kpi:
       KOBOCAT_PUBLIC_SUBDOMAIN: "kc"
       KOBOFORM_PUBLIC_SUBDOMAIN: "kobo"
       PUBLIC_REQUEST_SCHEME: "https"
+      # Celery configuration
+      CELERY_WORKER_AUTOSCALE: "2,3"
+      CELERY_WORKER_MAX_TASKS_PER_CHILD: "10000"
```

Comment:
Why do we just handle KPI? What about KoBoCAT? RestService and Legacy exports could overwhelm the workers if they don't scale too.

Reply:
We already can override kc, as the command doesn't specify concurrency. However we could use the same defaults for consistency.